### DISCUSSION TITLE
Need help to run the application

### DISCUSSION THREAD

**Discussion by makrony1** [simplcommerce/SimplCommerce#1028] [Q&A]
Hello, I am trying to run the application, after build the project from visual studio 2019, when I try to run it, it shows an error page saying the 
`InvalidOperationException: The view 'Index' was not found. The following locations were searched: /Areas/Core/Views/Home/Index.en-US.cshtml /Areas/Core/Views/Home/Index.en.cshtml /Areas/Core/Views/Home/Index.cshtml /Areas/Core/Views/Shared/Index.en-US.cshtml /Areas/Core/Views/Shared/Index.en.cshtml /Areas/Core/Views/Shared/Index.cshtml /Views/Shared/Index.en-US.cshtml /Views/Shared/Index.en.cshtml /Views/Shared/Index.cshtml /Pages/Shared/Index.en-US.cshtml /Pages/Shared/Index.en.cshtml /Pages/Shared/Index.cshtml`

if anyone can help me with this that would be very nice of you.

**Comment by technowriter** [comment]
Run it in Vs2022
